DESCRIPTION

```diff
@@ -44,7 +44,7 @@ Remotes:
     tidymodels/parsnip
 Config/Needs/website:
     tidymodels,
-    tidymodels/parsnip,
+    tidymodels/parsnip@set-mode-censored,
```

Comment:
I'll merge the parsnip PR as soon as CI is done you you can get rid of the branch specification.

Reply:
in the meantime, I'll put it in the correct place (aka Remotes) :see_no_evil: :smile: